src/python/src/grpc/_adapter/_low_test.py

```diff
@@ -115,6 +115,10 @@ def tearDown(self):
   def _perform_echo_test(self, test_data):
     method = 'test method'
     details = 'test details'
+    server_metadata_key = 'my_server_key'
```

Comment:
In the server->client direction there is both "leading" metadata ("headers") and "trailing" metadata ("trailers") - please exercise both kinds in this test (so, along with client->server metadata, three separate metadata values in total).